crates/tuono/tests/cli_build.rs

```diff
@@ -200,3 +200,72 @@ fn build_fails_with_no_config() {
         .failure()
         .stderr("Cannot find tuono.config.ts - is this a tuono project?\n");
 }
+
+#[test]
+#[serial]
+fn it_inits_new_git_repo_by_default_with_git_installed() {
+    let temp_tuono_project = TempTuonoProject::new();
```

Comment:
The same problem in the unit test happens here (if the host machine has not installed `git` the tests fail).
We should consider mocking it, otherwise we gonna increase test complexity and flakiness